src/ralph/sim_cards/models.py

```diff
@@ -0,0 +1,120 @@
+from django.conf import settings
+from django.core.validators import (
+    MaxLengthValidator,
+    MinLengthValidator,
+    RegexValidator
+)
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.back_office.models import BackOfficeAssetStatus, Warehouse
+from ralph.lib.mixins.models import (
+    AdminAbsoluteUrlMixin,
+    NamedMixin,
+    TimeStampMixin
+)
+from ralph.lib.transitions.fields import TransitionField
+
+
+class CellularCarrier(AdminAbsoluteUrlMixin, NamedMixin, models.Model):
+    pass
+
+
+class SIMCard(AdminAbsoluteUrlMixin, TimeStampMixin, models.Model):
+    pin1 = models.CharField(
+        max_length=8, null=True, blank=True,
```

Comment:
try to remove ``max_length=8`` or ``MaxLengthValidator(8)``

Reply:
but `max_length` here will affect database field, whether `MaxLengthValidator` will affect only data saved through the model. I'll keep `max_length` here as well

Reply:
done, removed MaxLengthValidator